src/components/deploymentConfig/DeploymentTemplateView/DeploymentTemplateView.component.tsx

```diff
@@ -296,16 +313,153 @@ export const renderEditorHeading = (
                         className={`cursor ${overridden ? 'cr-5' : 'cb-5'}`}
                         onClick={handleOverride}
                     >
-                        {overridden ? 'Delete override' : 'Allow override'}
+                        {isValues?(overridden ? 'Delete override' : 'Allow override'):''}
                     </span>
                 )}
             </div>
         </div>
     )
-}
 
-export const SuccessToastBody = ({ chartConfig }) => {
+export const CompareWithApprovalPendingAndDraft = ({
+    isEnvOverride,
+    overridden,
+    readOnly,
+    environmentName,
+    selectedChart,
+    handleOverride,
+    latestDraft,
+    isPublishedOverriden,
+    isDeleteDraftState,
+    setShowDraftData,
+    isValues
+}: CompareWithApprovalPendingAndDraftProps) => {
+    const [selectedOption, setSelectedOption] = useState({ id: 0, label: 'Approval Pending' })
+
+    const options = [
+        {
+            label: 'Manifest generated from',
+            options: [
+                { id: 0, label: 'Approval Pending' },
+                { id: 1, label: `${isValues?'Values':'Manifest'} from draft` },
+            ],
+        },
+    ]
+
+    useEffect(() => {
+        setShowDraftData(selectedOption.id === 1)
+    }, [selectedOption])
+
+
+    const onChange = (selected) => {
+        setSelectedOption(selected)
+    }
+
+    const formatLabel = (option) => (
+            <div className="flex left column">
+                <span className="w-100 dc__ellipsis-right">{option.label}</span>
+            </div>
+        )
+
     return (
+        <div className="flex dc__content-space w-100">
+            <div className="flex left">
+                {!readOnly && <Edit className="icon-dim-16 mr-10" />}
+                {latestDraft ? (
+                    <span className="fw-6 mr-4">
+                        <ReactSelect
+                            options={options}
+                            isMulti={false}
+                            value={selectedOption}
+                            // defaultValue={selectedOption.options[0]}
+                            isOptionSelected={(option, selected) => option === selected[0]}
+                            classNamePrefix="compare-template-values-select"
+                            formatOptionLabel={formatLabel}
+                            // isOptionDisabled={(option) => option.value === 0}
+                            isSearchable={false}
+                            onChange={onChange}
+                            components={{
+                                IndicatorSeparator: null,
+                                Option,
+                                DropdownIndicator,
+                                ValueContainer: customValueContainer,
+                            }}
+                            styles={{
```

Comment:
We should move this style to some constant or use some existing style, if aaplicable

Reply:
done